spec/javascript/components/applications/__snapshots__/ApplicationPage.test.js.snap

```diff
@@ -183,7 +183,7 @@ Array [
                     Created By
                   </h4>
                   <p>
-                    Prod Vertiba
+                    None
```

Comment:
not sure if we want this change

Reply:
this bit should go away with my next push